tests/installation/bootloader_s390.pm

```diff
@@ -15,6 +15,7 @@
 use base "installbasetest";
 
 use testapi;
+use opensusebasetest;
```

Comment:
this module ISA installbasetest which ISA opensusebasetest so I do not get why this should be needed